src/app/view/components/dot-container-selector/dot-container-selector.component.spec.ts

```diff
@@ -156,55 +142,6 @@ describe('ContainerSelectorComponent', () => {
         expect(paginatorService.filter).toEqual(filter);
     }));
 
-    it('should add containers to containers list and emit a change event', () => {
```

Comment:
why ?

Reply:
this component doesn't depend on `data` any more